statistics.md

```diff
@@ -6,97 +6,52 @@ aliases: ['/docs/dev/statistics/','/docs/dev/reference/performance/statistics/']
 
 # Introduction to Statistics
 
-TiDB uses statistics to decide [which index to choose](/choose-index.md).
+Tidb uses statistics as input to the optimizer to estimate the number of rows processed in each plan step for a SQL statement. The optimizer estimates the cost of each available plan choice - including [index accesses](/choose-index.md), the sequence of table joins, and produces a cost for each available plan. The optimizer then picks the execution plan with the lowest overall cost.
 
-## Versions of statistics
-
-The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`.
-
-- For TiDB Self-Hosted, the default value of this variable changes from `1` to `2` starting from v5.3.0.
```

Comment:
We could do some `<CustomContent platform="tidb-cloud">` here, but I don't think we should.

Reply:
This custom content is existing. I can certainly remove it if that is preferred.